packages/icon/src/react/icons/index.js

```diff
@@ -13,63 +25,132 @@ export { CaretDownIcon } from './CaretDownIcon.dist.js'
 export { CaretLeftIcon } from './CaretLeftIcon.dist.js'
 export { CaretRightIcon } from './CaretRightIcon.dist.js'
 export { CaretUpIcon } from './CaretUpIcon.dist.js'
+export { CcIcon } from './CcIcon.dist.js'
 export { ChannelAddIcon } from './ChannelAddIcon.dist.js'
 export { ChannelIcon } from './ChannelIcon.dist.js'
 export { ChatIcon } from './ChatIcon.dist.js'
 export { CheckCircleFillIcon } from './CheckCircleFillIcon.dist.js'
+export { CheckCircleIcon } from './CheckCircleIcon.dist.js'
 export { CheckIcon } from './CheckIcon.dist.js'
 export { ClockIcon } from './ClockIcon.dist.js'
+export { ClockWarningIcon } from './ClockWarningIcon.dist.js'
 export { CloseCaptionedFillIcon } from './CloseCaptionedFillIcon.dist.js'
 export { CloseIcon } from './CloseIcon.dist.js'
+export { CloudIcon } from './CloudIcon.dist.js'
+export { CloudOffIcon } from './CloudOffIcon.dist.js'
 export { CodeBrandedIcon } from './CodeBrandedIcon.dist.js'
 export { CodeIcon } from './CodeIcon.dist.js'
+export { ConnectionWarningIcon } from './ConnectionWarningIcon.dist.js'
 export { CopyIcon } from './CopyIcon.dist.js'
+export { CoursesIcon } from './CoursesIcon.dist.js'
+export { CurriculumIcon } from './CurriculumIcon.dist.js'
 export { DashboardIcon } from './DashboardIcon.dist.js'
+export { DesktopIcon } from './DesktopIcon.dist.js'
 export { DownloadIcon } from './DownloadIcon.dist.js'
+export { EnvelopeArrowIcon } from './EnvelopeArrowIcon.dist.js'
 export { EnvelopeIcon } from './EnvelopeIcon.dist.js'
+export { ExternalLinkIcon } from './ExternalLinkIcon.dist.js'
+export { EyeIcon } from './EyeIcon.dist.js'
+export { EyeOffIcon } from './EyeOffIcon.dist.js'
 export { FileIcon } from './FileIcon.dist.js'
+export { FilterIcon } from './FilterIcon.dist.js'
+export { FullscreenExitIcon } from './FullscreenExitIcon.dist.js'
+export { FullscreenIcon } from './FullscreenIcon.dist.js'
 export { GearIcon } from './GearIcon.dist.js'
+export { GlobeIcon } from './GlobeIcon.dist.js'
 export { HandIcon } from './HandIcon.dist.js'
+export { HelpIcon } from './HelpIcon.dist.js'
+export { HistoryIcon } from './HistoryIcon.dist.js'
+export { HomeAdminToolsIcon } from './HomeAdminToolsIcon.dist.js'
+export { HomeContentToolsIcon } from './HomeContentToolsIcon.dist.js'
 export { HomeIcon } from './HomeIcon.dist.js'
 export { InfoIcon } from './InfoIcon.dist.js'
 export { InteractiveIcon } from './InteractiveIcon.dist.js'
+export { IqIcon } from './IqIcon.dist.js'
 export { LinkIcon } from './LinkIcon.dist.js'
 export { ListIcon } from './ListIcon.dist.js'
+export { LocationIcon } from './LocationIcon.dist.js'
+export { LockFillIcon } from './LockFillIcon.dist.js'
 export { LockIcon } from './LockIcon.dist.js'
-export { LogoutIcon } from './LogoutIcon.dist.js'
```

Comment:
This will require a BREAKING CHANGE in the commit to gen a new major version on release.

Reply:
Done!